Dockerfile

```diff
@@ -86,7 +86,8 @@ RUN set -eux ; \
       zlib \
       diffutils \
       krb5-workstation \
-      fuse ; \
+      fuse \
+      openssl ; \
```

Comment:
Originally this list is in ABC order, but seems it was recently "broken" a bit.  Please add the new package between `nmap-ncat` and `python3`.

Reply:
Thanks for the info, after this change is merged I'm going to create a small jira just to fix this alphabetical order.